tests/integration_tests/test_chat_completion.py

```diff
@@ -0,0 +1,90 @@
+import re
+from typing import List
+
+import pytest
+
+from tests.conftest import TEST_DEPLOYMENTS_CONFIG_PATH
+from tests.integration_tests.base import (
+    TestCase,
+    TestDeployments,
+    TestSuite,
+    TestSuiteBuilder,
+)
+from tests.integration_tests.chat_completion_suites.text import (
+    text_common,
+    text_databricks_excluded,
+    text_mistral_excluded,
+)
+from tests.integration_tests.chat_completion_suites.tools import tools_common
+from tests.integration_tests.chat_completion_suites.vision import vision_common
+from tests.utils.openai import (
+    ChatCompletionResult,
+    ExpectedException,
+    chat_completion,
+)
+
+
+def create_test_cases(
+    test_case_builder: TestSuiteBuilder,
+) -> List[TestCase]:
+
+    return [
+        test_case
+        for streaming in (False, True)
+        # for streaming in (True, False)
+        for deployment in TestDeployments.from_config(
```

Comment:
`TestDeployments.from_config(TEST_DEPLOYMENTS_CONFIG_PATH)` is repeated twice in the codebase. Maybe it is worth declaring it as a helper?

Reply:
I don't see a big need for this, since it's just a one-liner

Reply:
It's not about code repetition, but about computing the same thing only once, because 
`TestDeployments.from_config(TEST_DEPLOYMENTS_CONFIG_PATH)` is a test-wide constant, so the right place for it would be alongside `TEST_DEPLOYMENTS_CONFIG_PATH` in `constants.py`:

```python
TEST_DEPLOYMENTS_CONFIG_PATH = ...
TEST_DEPLOYMENT_CONFIG = TestDeployments.from_config(TEST_DEPLOYMENTS_CONFIG_PATH)
```

Reply:
Ok, added it to constants, though it required to add this part

```python
try:
    TEST_DEPLOYMENT_CONFIG = TestDeployments.from_config(
        TEST_DEPLOYMENTS_CONFIG_PATH
    )
except FileNotFoundError:
    TEST_DEPLOYMENT_CONFIG = TestDeployments(
        deployments=[], app_config=ApplicationConfig()
    )
```
To handle cases in CI, where there is no config file in unit tests